pkg/kubelet/kuberuntime/helpers_linux.go

```diff
@@ -44,21 +49,22 @@ func milliCPUToShares(milliCPU int64) int64 {
 }
 
 // milliCPUToQuota converts milliCPU to CFS quota and period values
-func milliCPUToQuota(milliCPU int64) (quota int64, period int64) {
+func milliCPUToQuota(milliCPU int64, period int64) (quota int64) {
 	// CFS quota is measured in two values:
 	//  - cfs_period_us=100ms (the amount of time to measure usage across)
 	//  - cfs_quota=20ms (the amount of cpu time allowed to be used across a period)
 	// so in the above example, you are limited to 20% of a single CPU
 	// for multi-cpu environments, you just scale equivalent amounts
+	// see https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt for details
 	if milliCPU == 0 {
 		return
 	}
-
-	// we set the period to 100ms by default
-	period = quotaPeriod
+	if !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUCFSQuotaPeriod) {
+		period = quotaPeriod
```

Comment:
@szuecs that's precise place which worries me: previously we used `quotaPeriod` as-is, which, translated to `time.Duration`, equals to 100 microseconds. And now we set it to `cpuCFSQuotaPeriod`, which according to code and comments is 100 millisecond default.

Reply:
1. I don't see how this code can be worrying, because it shows the same assignment as before. If not the feature gate is set, set it to quotaPeriod as before. see https://github.com/kubernetes/kubernetes/pull/63437/files/588d2808b77d11f235b6eba5c21bcaa89a2f7804#r932625167
2. if you enable the featuregate you can set to the value you like

Reply:
"ms" mentioned across the code alongside the tests with milliseconds and not microseconds confused me. Now I see it's still microseconds, and comments should be altered, and maybe tests as well. I'll prepare the PR, thanks for the help!

Reply:
Thanks @paskal , makes sense to update, if it's wrong :)

Reply:
I've created #111520 about it.

Reply:
I was wrong, and the original code was correct. 100ms is indeed a default value in Linux, and I got confused by conversions in two places in code that had no comments about them. PR #112123 adds comments to the code added here.

Reply:
I think I read the kernel documentation at least 5 times when I did investigation and the code here.